bin/Cargo.toml

```diff
@@ -102,6 +105,9 @@ hickory-resolver.workspace = true
 test-support.workspace = true
 toml.workspace = true
 webpki-roots.workspace = true
+# metrics tests
+reqwest = { version = "0.12.12", default-features = false, features = [] }
+prometheus-parse = { version = "0.2.5" }
```

Comment:
These version numbers should probably be declared in the workspace Cargo.toml as well

Reply:
Will move that to the top level.